torchtune/data/_messages.py

```diff
@@ -393,10 +393,10 @@ def __call__(self, sample: Mapping[str, Any]) -> Mapping[str, Any]:
         return {"messages": messages}
 
 
-class JSONToMessages(Transform):
+class OpenAIToMessages(Transform):
     """
-    Convert a single chat sample with identical json structure to torchtune's :class:`~torchtune.data.Message`
-    structure. This transform simply creates Message dataclasses from the provided jsons.
+    Convert a single chat sample adhering to the `OpenAI chat completion <https://platform.openai.com/docs/api-reference/chat>`_
+    json structure to torchtune's :class:`~torchtune.data.Message` structure.
```

Comment:
```suggestion
    JSON structure to torchtune's :class:`~torchtune.data.Message` structure.
```

Reply:
at this point it should be a real word in the dictionary